src/client/containers/OrderConfirmationPage/OrderConfirmationPage.styles.css

```diff
@@ -1,3 +1,57 @@
-div {
-  color: #687808;
+.confirmation-page {
+    align-items: center;
+    margin: 4%;
 }
+
+.confirmation-page .inner-div {
+    background-color: #F6F8EB;
+}
+
+.confirmation-page .wrapper2 {
+    text-align: center;
+    margin-right: 30%;
+}
+
+.confirmation-page .wrapper {
+    width: 90%;
+    text-align: center;
+    padding-top: 8%;
+}
+
+.confirmation-page .wrapper-content {
+    width: 90%;
+    font-size: 1.2rem;
+    padding: 4rem;
+    margin-left: auto;
+    background-color: #fff;
+    color: black;
+}
+
+.confirmation-page .btn {
```

Comment:
Avoid short names like `.btn`, and `button` isn't much better when compared to `button2`.
Describe what these buttons do! 
`to-start-button`? `continue-shopping-button`?

And for `btn2`, simply `review-order-button` would suffice.